neuraxle/steps/numpy.py

```diff
@@ -121,3 +122,19 @@ def _print(self, data_inputs):
 
     def _print_one(self, data_input):
         print(self.__class__.__name__ + " (one):", data_input.shape, self.custom_message)
+
+class MultiplyByN(NonFittableMixin, BaseStep):
```

Comment:
I would love that you use that in the AutoML sleep test, and using many `Sleeper()` before each checkpoint and/or at other places int the pipeline..